README.md

```diff
@@ -1,2 +1,3 @@
 # ISSAC-FE
-# version 1.0
\ No newline at end of file
+# version 1.0
```

Comment:
please change the 
`Hello world `
to 
`Hello Michael`

Reply:
ok